stix2viz/stix2viz/stix2viz.js

```diff
@@ -788,15 +867,40 @@ function makeEdgeObject(sourceRef, targetRef, label, stixId=null)
  * @param stixObject The STIX object.  Provided in case any info from it is
  *      needed for configuring the node
```

Comment:
The function signature changed, and now the doc comment is incomplete.  If there is a doc comment, it would be nice to keep it up to date.

Reply:
done